install/compilation_options.md

```diff
@@ -0,0 +1,247 @@
+$> chmod +x compile_boost.sh
+$> ./compile_boost.sh
+```
+
+Version 1_69_0 was not tested on Centos6.4 but 1_59_0 is working well.
+
+## GoogleTest
+
+[GoogleTest website](https://github.com/google/googletest)
+
+``` bash
+$> echo 'GTEST_VERSION=1.8.1
+GTEST_ARCHIVE=release-${GTEST_VERSION}.tar.gz
+GTEST_DIRECTORY=googletest-release-$GTEST_VERSION
+GTEST_DOWNLOAD_URL=https://github.com/google/googletest/archive
+SCRIPT_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+BUILD_DIR=$SCRIPT_DIR/googletest/build/$GTEST_VERSION
+BUILD_TYPE=Release
+INSTALL_DIR=$SCRIPT_DIR/googletest/install/$GTEST_VERSION
+C_COMPILER=$(command -v gcc)
+CXX_COMPILER=$(command -v g++)
+CXX11_ENABLED=NO
+NB_PROCESSORS_USED=1
+cd $SCRIPT_DIR
+if [ ! -f "${GTEST_ARCHIVE}" ]; then
+  if [ -x "$(command -v wget)" ]; then
+    wget ${GTEST_DOWNLOAD_URL}/${GTEST_ARCHIVE} || { echo "Error while downloading GoogleTest."; exit 1; }
+  else
+    echo "You need to install wget."
+    exit 1
+  fi
+fi
+if [ ! -d "$SCRIPT_DIR/$GTEST_DIRECTORY" ]; then
+  tar -xzf $GTEST_ARCHIVE -C $SCRIPT_DIR || { echo "Error while extracting GoogleTest."; exit 1; }
+fi
+if [ ! -d "$BUILD_DIR" ]; then
+  mkdir -p $BUILD_DIR
+fi
+if [ ! -d "$INSTALL_DIR" ]; then
+  mkdir -p $INSTALL_DIR
+fi
+if [ "$(echo "$CXX11_ENABLED" | tr "[:upper:]" "[:lower:]")" = "no" -o "$(echo "$CXX11_ENABLED" | tr "[:upper:]" "[:lower:]")" = "false" -o "$(echo "$CXX11_ENABLED" | tr "[:upper:]" "[:lower:]")" = "off" ]; then
+  CXX_STD_FLAG="98"
+else
+  CXX_STD_FLAG="11"
+fi
+cd $BUILD_DIR
+cmake "$SCRIPT_DIR/$GTEST_DIRECTORY" -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX=$INSTALL_DIR -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DCMAKE_C_COMPILER=$C_COMPILER -DCMAKE_CXX_COMPILER=$CXX_COMPILER -DCMAKE_CXX_FLAGS="-std=c++$CXX_STD_FLAG" -DCVF_VERSION:STRING=1.9.0 || { echo "Error while cmake configuration of GoogleTest."; exit 1; }
+make -j $NB_PROCESSORS_USED || { echo "Error while make."; exit 1; }
+make install || { echo "Error while make install."; exit 1; }' > compile_googletest.sh
+$> chmod +x compile_googletest.sh
+$> ./compile_googletest.sh
+```
+
+`DCVF_VERSION` flag might be unused on newer CMake version, it is only here because we had trouble with cmake version 2.8.12.2 (at least).
+
+**Warnings** Version 1.8.1 does not work on Centos6.4 ([Related Issue](https://github.com/google/googletest/pull/2073)) and 1.8.0 causes problems with thread on runtime so we recommend to use the system one (1.5.0) even if at the time GoogleTest instructed to not use pre-built libraries. For more recent OS this procedure and the use of the latest version of GoogleTest should be fine. On Debian based OS **libgtest-dev** provides sources but not compiled version as instructed by GoogleTest so we recommend to install it with the previous procedure (tested on Debian 9). If you have any trouble building GoogleTest stackoverflow is full of answers for various problems on various OS, otherwise don't hesitate to ask us.
```

Comment:
add rte-des-simulation-dynamique@rte-france.com in () after don't hesitate to ask us